specs/phase0/beacon-chain.md

```diff
@@ -603,18 +607,50 @@ def bytes_to_uint64(data: bytes) -> uint64:
 
 #### BLS Signatures
 
+##### IETF standard
+
 Eth2 makes use of BLS signatures as specified in the [IETF draft BLS specification draft-irtf-cfrg-bls-signature-03](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-03). Specifically, eth2 uses the `BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_` ciphersuite which implements the following interfaces:
 
-- `def Sign(SK: int, message: Bytes) -> BLSSignature`
-- `def Verify(PK: BLSPubkey, message: Bytes, signature: BLSSignature) -> bool`
+- `def Sign(SK: int, message: bytes) -> BLSSignature`
 - `def Aggregate(signatures: Sequence[BLSSignature]) -> BLSSignature`
-- `def FastAggregateVerify(PKs: Sequence[BLSPubkey], message: Bytes, signature: BLSSignature) -> bool`
-- `def AggregateVerify(PKs: Sequence[BLSPubkey], messages: Sequence[Bytes], signature: BLSSignature) -> bool`
+- `def _Verify(PK: BLSPubkey, message: bytes, signature: BLSSignature) -> bool`
+- `def _AggregateVerify(PKs: Sequence[BLSPubkey], messages: Sequence[bytes], signature: BLSSignature) -> bool`
+- `def _FastAggregateVerify(PKs: Sequence[BLSPubkey], message: bytes, signature: BLSSignature) -> bool`
 
 Within these specifications, BLS signatures are treated as a module for notational clarity, thus to verify a signature `bls.Verify(...)` is used.
 
 *Note*: The non-standard configuration of the BLS and hash to curve specs is temporary and will be resolved once IETF releases BLS spec draft 3.
 
+##### Eth2 BLS wrappers
+
+We use the following Eth2-specific BLS wrappers to be compatible with IETF BLS standard.
+
+```python
+def is_zero_sk(PK: int) -> bool:
```

Comment:
Do you mean `is_zero_pk`? Also, we have to be consistent here. You're using both "zero" and "infinity". My suggestion would be to use "zero" or "identity" everywhere (the word "infinity" can be needlessly confusing).